chemie/merch/views.py

```diff
@@ -0,0 +1,114 @@
+from django.shortcuts import render
+from .forms import MerchForm, MerchCategoryForm, SortingForm
+from django.core.paginator import Paginator
+from django.contrib.auth.decorators import permission_required, login_required
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+from django.shortcuts import get_object_or_404
+from .models import Merch, MerchCategory
+from django.contrib import messages
+from dal import autocomplete
+from django.utils.html import format_html
+
+# Create your views here.
+@permission_required("merch.add_merch")
+def create_merch(request):
+    form = MerchForm(request.POST or None, request.FILES or None)
+
+    if request.POST:
+        if "another" in request.POST and form.is_valid():
+            instance = form.save(commit=False)
+            instance.save()
+            return HttpResponseRedirect(reverse("merch:create"))
+        if form.is_valid():
+            instance = form.save(commit=False)
+            instance.save()
+            return HttpResponseRedirect(reverse("merch:index"))
+
+    context = {"form": form}
+
+    return render(request, "merch/create_merch.html", context)
+
+
+@permission_required("merch.add_merch")
+def create_category(request):
+    form = MerchCategoryForm(request.POST or None, request.FILES or None)
+
+    if request.POST:
+        if "another" in request.POST and form.is_valid():
+            instance = form.save(commit=False)
+            instance.save()
+            return HttpResponseRedirect(reverse("merch:create_category"))
+
+        if form.is_valid():
+            instance = form.save(commit=False)
+            instance.save()
```

Comment:
Why is commit=False when instance is saved in the next line?
Could be reduced to simply
`if form.is_valid(): \
    form.save() \
    return ...`